src/herbpy/action/rogue.py

```diff
@@ -0,0 +1,359 @@
+    pointing_coord = GetPointFrom(env, focus)
+    return Point(robot, pointing_coord, manip, render)
+
+@ActionMethod
+def PresentAt(robot, focus, manip=None, render=True):
+    """
+    @param robot The robot performing the presentation
+    @param focus The 3-D coordinate in space or object that 
+                 is being presented
+    @param manip The manipulator to perform the presentation with. 
+                 This must be the right arm.
+    @param render Render tsr samples during planning
+    """
+    env = robot.GetEnv()
+    presenting_coord = GetPointFrom(env, focus)
+    return Present(robot, presenting_coord, manip, render)
+
+@ActionMethod
+def SweepAt(robot, start, end, manip=None, margin=0.3, render=True):
+    """
+    @param robot The robot performing the sweep
+    @param start The object or 3-d position that marks the start
+    @param end The object of 3-d position that marks the end
+    @param manip The manipulator to perform the sweep
+    @param margin The distance between the start object and the hand,
+                  so the vertical space between the hand and objects. 
+                  This must be enough to clear the objects themselves.
+    @param render Render tsr samples during planning
+    """
+    env = robot.GetEnv()
+    start_coord = GetPointFrom(env, start)
+    end_coord = GetPointFrom(env, end)
+    return Sweep(robot, start_coord, end_coord, manip, margin, render)
+
+
+def GetPointFrom(env, focus):
+    """
+    @param env The environment where the item exists
+    @param focus The area to be referred to
+    """
+    #Pointing at an object
+    if isinstance(focus, (openravepy.KinBody, openravepy.KinBody.Link)):
+        with env: 
+            focus_trans = focus.GetTransform()
+        coord = list(focus_trans[0:3, 3])
+
+    #Pointing at a point in space as numpy array
+    elif (isinstance(focus, numpy.ndarray) and (focus.ndim == 1) 
+           and (len(focus) == 3)):
+        coord = list(focus)
+
+    #Pointing at point in space as 4x4 transform
+    elif isinstance(focus, numpy.ndarray) and (focus.shape == (4, 4)):
+        coord = list(focus[0:3, 3])
+
+    #Pointing at a point in space as list or tuple
+    elif (isinstance(focus, (tuple, list)) and len(focus) == 3):
+        coord = focus
+
+    else:
+        raise prpy.exceptions.PrPyException('Focus of the point is an \
+                unknown object')
+
+    return coord
+
+def Point(robot, coord, manip=None, render=False):
+    """
+    @param robot The robot performing the point
+    @param focus The 3-D coordinate in space that is being pointed at
+    @param manip The manipulator to perform the point with. 
+                 This must be the right arm
+    @param render Render tsr samples during planning
+    """
+    if manip is None:
+        manip = robot.right_arm
+
+    if manip.GetName() != 'right':
+        raise prpy.exceptions.PrPyException('Pointing is only defined \
+                on the right arm.')
```

Comment:
This will insert a bunch of spaces into the comment. You should continue the line like this:
```python
raise prpy.exceptions.PrPyException('Pointing is only defined'
                                    'on the right arm.')
```